common/source/docs/common-mamba405-mk2.rst

```diff
@@ -0,0 +1,112 @@
+================================
+Mamba F405 MK2 Flight Controller
+================================
+
+.. image:: ../../../images/mamba_f405mk2_board.jpg
+     :target: ../_images/mamba_f405mk2_board.jpg
+
+https://www.diatone.us/products/mamba-f405-flight-controller-mk2
+
+The Mamba F405 MK2 is a flight controller produced by
+`Diatone <https://www.diatone.us>`__.
+
+.. note::
+
+   Support for this board is available with Plane 4.1.0, and higher
+
+Features
+========
+
+-  STM32F405RGT6 microcontroller
+-  MPU6000 IMU
+-  AT7456E OSD
+-  4 UARTs
+-  4 PWM outputs
+
+Pinout
+======
+
+.. figure:: ../../../images/mamba_f405mk2_pinout.png
+   :target: ../_images/mamba_f405mk2_pinout.png
+
+   Mamba F405 MK2 Board
+
+UART Mapping
+============
+
+======= ================ ================
+Name    Pin              Function
+======= ================ ================
+SERIAL0 COMPUTER         USB
+SERIAL1 PPM/RX1/SBUS/TX1 UART1 (RC Input)
+SERIAL3 TX3/RX3          UART3 (Telem1)
+SERIAL6 TX6/RX6          UART6 (GPS)
+======= ================ ================
+
+RC Input
+========
+
+RC input is configured on the PPM (UART1_RX) pin. It supports all RC
```

Comment:
I have doubts that this actually works with the hwdef as it stands...it appears that the inverter from SBUS to PPM is a controlled inverter and therefore would need to be actively tri-stated in order to attach anything to the PPM input...I have asked the author, but he never responded....so I dont want to post incorrect information until that is resolved...

Reply:
Aren't we using software serial for RC input?

Reply:
The way this hwdef is configured you need to use the SBUS input for all RC protocols , PPM input is unuseable since its driven by the SBUS inverter all the time and not disabled easily (can by forcing the GPIO)...and I am pretty sure that ALT config 1 does not work...it would need to also disable the inverter to use the UART1 RX input (PPM pin)